index.html

```diff
@@ -0,0 +1,95 @@
+              <li><a class="dropdown-item" href="#section1">Section 1</a></li>
+              <li><a class="dropdown-item" href="#section2">Section 2</a></li>
+              <li><a class="dropdown-item" href="#section3">Section 3</a></li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <div class="container">
+    <section id="welcome-section" class="bg-success">
+      <h1>
+        <div class="text-center">Welcome to my Personal Profile</div>
+      </h1>
+      <p class="lead px-4">Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellendus incidunt error provident unde inventore accusamus iure non reprehenderit consequatur voluptatem explicabo tempora repudiandae, enim nulla dolore laudantium temporibus. Inventore, iure?
+        Ratione et numquam assumenda nostrum labore maxime dolorem quia voluptatum veritatis consectetur mollitia, saepe nam adipisci quidem hic? Quam enim ab possimus laudantium doloribus ducimus maiores omnis repellat numquam aspernatur.
+        Ad deleniti consequuntur voluptatibus exercitationem magnam cupiditate corporis incidunt vel suscipit omnis sunt assumenda esse fuga, tempora deserunt a laudantium, eos quidem illo consectetur repellendus, similique mollitia ex totam! Cupiditate!
+        Illum ipsa, ipsam repellat similique ad, qui a eligendi obcaecati ab labore atque fugiat sequi cupiditate? Dolorem corporis iure veritatis numquam ullam sint incidunt non, illum dolorum excepturi aliquam quas!
+        Sequi quae voluptatibus, tempore nostrum praesentium, inventore cumque minus debitis at expedita aspernatur placeat tempora soluta excepturi dolorum eos corrupti quisquam? Cupiditate velit numquam aliquam vitae inventore harum ad autem!
+        Eius neque facere laboriosam, natus laborum iste reprehenderit suscipit ipsam odit dignissimos enim amet consequatur eaque tempore iusto delectus beatae nobis veniam, vero repellendus sit, numquam accusantium. Architecto, sit beatae?
+        Eligendi laborum excepturi nihil fuga reiciendis veniam quibusdam ipsam dolore consequuntur qui commodi incidunt officia rem facilis illo eum dolorum voluptates libero, quisquam cumque doloremque numquam porro atque rerum. Ex.
+        Esse nesciunt fugiat impedit reiciendis? Recusandae vero cupiditate, repudiandae quo quisquam inventore at temporibus nisi facere placeat dignissimos eveniet totam, obcaecati nihil suscipit! Iure, vero? Itaque consectetur possimus aperiam nobis!
+        Itaque aut voluptatem atque et vitae voluptates cupiditate veritatis ipsam totam veniam porro debitis, in tempore enim placeat accusamus ipsa vero, soluta repudiandae omnis voluptas non voluptatibus quis! Similique, nobis.
+        Alias nesciunt eos, ab aut sed quaerat perferendis ratione eum velit molestiae ex, est asperiores deleniti sequi odio laboriosam rem dolore incidunt repellendus explicabo. Esse sapiente non voluptatibus aliquid qui?</p>
+    </section>
+
+    <section id="projects">
```

Comment:
- [ ] It would be better if an empty space is not left in your code. Leaving empty spaces does not make your code to follow HTML && CSS best practices